cmd/cli/get.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+//NewCommandGet generate run cmd
+func NewCommandGet() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "Display one or many resources",
+		Long:  `list of resorces comannd can display includes: studies, study, trials, trial, models, model`,
+	}
+
+	//set local flag
+
+	//add subcommand
+	cmd.AddCommand(NewCommandGetStudies())
+	cmd.AddCommand(NewCommandGetStudy())
+	//	cmd.AddCommand(NewCommandGetTrials())
+	//	cmd.AddCommand(NewCommandGetTrial())
```

Comment:
I am wondering if we could enable the two command. WDYT

Reply:
Unfortunately, we don't have the get trial API in katib-manager.
I will refactor API and add low-level APIs (related #66 ). Then I will enable these commands.

Reply:
SGTM, I misunderstand the comments.